test/instance.js

```diff
@@ -176,6 +180,159 @@ describe('Bigtable/Instance', function() {
     });
   });
 
+  describe('createAppProfile', function() {
+    it('should provide the proper request options', function(done) {
+      instance.bigtable.request = function(config) {
+        assert.strictEqual(config.client, 'BigtableInstanceAdminClient');
+        assert.strictEqual(config.method, 'createAppProfile');
+
+        assert.strictEqual(config.reqOpts.parent, INSTANCE_ID);
+        assert.strictEqual(config.reqOpts.appProfileId, APP_PROFILE_NAME);
+
+        assert.strictEqual(config.gaxOpts, undefined);
+
+        done();
+      };
+
+      instance.createAppProfile(
+        APP_PROFILE_NAME,
+        {routing: 'any'},
+        assert.ifError
+      );
+    });
+
+    it('should throw if the routing option is not provided', function() {
+      assert.throws(
+        instance.createAppProfile.bind(null, APP_PROFILE_NAME, assert.ifError)
+      ),
+        /An app profile must contain a routing policy\./;
+    });
+
+    it('should accept gaxOptions', function(done) {
+      var options = {
+        routing: 'any',
+        gaxOptions: {},
+      };
+
+      instance.bigtable.request = function(config) {
+        assert.strictEqual(config.gaxOpts, options.gaxOptions);
+        done();
+      };
+
+      instance.createAppProfile(APP_PROFILE_NAME, options, assert.ifError);
+    });
+
+    describe('should respect the routing option when', function() {
```

Comment:
The sentence is incomplete.

Reply:
This is hard to do with bdd frameworks, it turns into something like:

should respect the routing option with
  an 'any' value  √
  a cluster value √

Reply:
ack